config/README.md

```diff
@@ -0,0 +1,119 @@
+# Dependencies installation
+
+For a faster creation of the environment (and Conda environments in general),
+you can also install [Mamba][mamba] on top of Conda. In that case, replace
+`conda` with `mamba` in the commands below (particularly in 
+`conda env create`).
+
+Create and activate the virtual environment with the required dependencies
+with Conda:
+
+```bash
+conda env create -f environment.yml
+conda activate mirflowz
+```
+
+If you plan to run _MIRFLOWZ_ via Conda, we recommend using the following
+command for a faster environment creation specially if you will run it on an
+HPC cluster.
+
+```bash
+conda config --set channel_priority strict
+```
+In addition, if you want to run _MIRFLOWZ_ via Singularity and do not already
+have it installed globally on your system, you must further update the Conda
+environment with:
+
+```bash
+conda env update -f environment.root.yml
+```
+
+> Mind that you must have the environment activated and root permissions on
+> your system to install Singularity.
+
+
+# Run the workflow on your own samples
+
+In order to run _MIRFLOWZ_ on your own samples, we recommend having all the
+input files inside a dedicated directory. This way is easier to keep the data
+together and reproduce an analysis. Assuming that your current directory is
+the repository's root directory, create a directory to store all your data
+and traverse to it with:
+
+```bash
+mkdir path/to/your_run
+cd path/to/your_run
+```
+
+### 1. Prepare the sample table
+
+Create an empty sample table. Refer to the
+[sample.tsv](../test/test_files/samples_table.tsv) test file to see what the
+table must look like or use it as a template.
+
+```bash
+touch samples.tsv
+```
+
+> Fill the sample table according to the following requirements:  
+>
+> - `sample`. This column contains the library name.  
+> - `sample_file`. In this column, you must provide the path to the library file.
+> The path must be relative to the working directory.  
+> - `adapter`.  This field must contain the adapter sequence in capital letters.  
+> - `format`. In this field you must state the library format. It can either be 
+> `fa` if providing a FASTA file or `fastq` if the library is a FASTQ file.  
+
+### 2. Prepare the genome resources
+
+There are 4 files you must provide: 
+
+1. A **`gzip`ped FASTA** file containing **reference sequences**, typically the
+   genome of the source/organism from which the library was extracted.
+
+2. A **`gzip`ped GTF** file with matching **gene annotations** for the
+   reference sequences above.
+
+> _MIRFLOWZ_ expects both the reference sequence and gene annotation files to
+> follow [Ensembl][ensembl] style/formatting. If you obtained these files from
+> a source other than Ensembl, you may first need to convert them to the
+> expected style to avoid issues!
```

Comment:
Would be cool if you found a reference to point people to, like a tool or a tutorial or something like that. But don't break your neck over it.

Reply:
Also, I would formulate it stronger:

"If you obtained these files from a source other than Ensembl, you must ensure that they adhere to the expected format by converting them, if necessary."

Same below.